cmd/aws-vpc-cni-init/main.go

```diff
@@ -0,0 +1,195 @@
+				return errors.Wrap(err, "Failed to disable tcp_early_demux")
+			}
+		} else {
+			err = sysctlUtil.Set(entry, 1)
+			if err != nil {
+				return errors.Wrap(err, "Failed to enable tcp_early_demux")
+			}
+		}
+		val, _ = sysctlUtil.Get(entry)
+		log.Infof("Updated %s to %d", entry, val)
+	}
+	return nil
+}
+
+func configureIPv6Settings(sysctlUtil sysctl.Interface, primaryIF string) error {
+	var err error
+	// Enable IPv6 when environment variable is set
+	// Note that IPv6 is not disabled when environment variable is unset. This is omitted to preserve default host semantics.
+	enableIPv6 := getEnv(envEnableIPv6, "false")
+	if enableIPv6 == "true" {
+		entry := "net/ipv6/conf/all/disable_ipv6"
+		err = sysctlUtil.Set(entry, 0)
+		if err != nil {
+			return errors.Wrap(err, "Failed to set disable_ipv6 to 0")
+		}
+		val, _ := sysctlUtil.Get(entry)
+		log.Infof("Updated %s to %d", entry, val)
+
+		entry = "net/ipv6/conf/all/forwarding"
+		err = sysctlUtil.Set(entry, 1)
+		if err != nil {
+			return errors.Wrap(err, "Failed to enable ipv6 forwarding")
+		}
+		val, _ = sysctlUtil.Get(entry)
+		log.Infof("Updated %s to %d", entry, val)
+
+		entry = "net/ipv6/conf/" + primaryIF + "/accept_ra"
+		err = sysctlUtil.Set(entry, 2)
+		if err != nil {
+			return errors.Wrap(err, "Failed to enable ipv6 accept_ra")
+		}
+		val, _ = sysctlUtil.Get(entry)
+		log.Infof("Updated %s to %d", entry, val)
+	}
+	return nil
+}
+
+func main() {
+	os.Exit(_main())
+}
+
+func _main() int {
+	log.Debug("Started Initialization")
+	pluginBins := []string{"loopback", "portmap", "bandwidth", "aws-cni-support.sh"}
+	var err error
+	for _, plugin := range pluginBins {
+		if _, err = os.Stat(plugin); err != nil {
+			log.WithError(err).Fatalf("Required executable: %s not found", plugin)
+			return 1
+		}
+	}
+
+	log.Infof("Copying CNI plugin binaries ...")
+	hostCNIBinPath := getEnv(envHostCniBinPath, defaultHostCNIBinPath)
+	err = cp.InstallBinaries(pluginBins, hostCNIBinPath)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to install binaries")
+		return 1
+	}
+	log.Infof("Copied all CNI plugin binaries to %s", hostCNIBinPath)
+
+	var primaryIF string
+	primaryIF, err = getNodePrimaryIF()
+	if err != nil {
+		log.WithError(err).Errorf("Failed to get primary IF")
+		return 1
+	}
+	log.Infof("Found primaryIF %s", primaryIF)
+
+	sysctlUtil := sysctl.New()
+	err = configureSystemParams(sysctlUtil, primaryIF)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to configure system parameters")
+		return 1
+	}
+
+	err = configureIPv6Settings(sysctlUtil, primaryIF)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to configure IPv6 settings")
+		return 1
+	}
+
+	// TODO: In order to speed up pod launch time, VPC CNI init container is not a Kubernetes init container.
+	// The VPC CNI container blocks on the existence of vpcCniInitDonePath
+	//err = cp.TouchFile(vpcCniInitDonePath)
+	//if err != nil {
+	//	log.WithError(err).Errorf("Failed to set VPC CNI init done")
```

Comment:
Lets delete these lines.

Reply:
These are going to be uncommented for #2137 , so I was leaving them to make it easier for that PR to rebase

Reply:
I see, sure.

Reply:
This will make it harder since it would in theory cause a conflict as opposed to a net new addition in the follow on PR

Reply:
These files do not exist in the new PR, so will not cause any conflict, and the entrypoint scripts the new PR is modifying are deleted by this PR, so will cause a conflict, but that was unavoidable and the resolution is easy (accept delete of file). #2137 needs cleanup anyway before it can be merged, so the goal is for the diff in these files to just be the uncommented lines

Reply:
Ok, that's just a bit odd but ok